pkg/validation/app.go

```diff
@@ -536,3 +535,21 @@ func contains(s []string, e string) bool {
 	}
 	return false
 }
+
+func isClusterSingleton(entry v1alpha1.AppCatalogEntry, cr, app v1alpha1.App) bool {
+	if entry.Spec.Restrictions.ClusterSingleton {
+		if !key.IsInOrgNamespace(cr) {
+			return true
+		} else if key.IsInOrgNamespace(cr) &&
```

Comment:
```suggestion
		} else if 
```

Reply:
The previous condition checks if isOrgNamepace is false so if you are this the else you already know its true